### ISSUE TITLE
Always provide accessible dialog heading

### ISSUE THREAD

**Issue by chris-steele** [adaptlearning/adapt-contrib-tutor#74]
### Subject of the issue/enhancement/features
If no visual title is required an accessible default should be provided. Related to https://github.com/adaptlearning/adapt-contrib-core/issues/173.

### Your environment
* all framework versions, devices and browsers

### Steps to reproduce
Remove `_feedback.title`, component `title` and `displayTitle` and note the absence of a heading for AT.

### Expected behaviour
AT should provide the user with a heading for the dialog to describe its purpose.

### Actual behaviour
No heading is read out in the absence of a visual title.

**Comment by oliverfoster** [comment]
I disagree. The default will never be more meaningful that the `role=dialog` tag.
